assets/js/payment-method-extensions/payment-methods/stripe/credit-card/payment-method.js

```diff
@@ -12,6 +12,7 @@ import { InlineCard, CardElements } from './elements';
 import { Elements, useElements, useStripe } from '@stripe/react-stripe-js';
 import { useState } from '@wordpress/element';
 import { __ } from '@wordpress/i18n';
+import { PaymentMethodIcons } from '@woocommerce/base-components/cart-checkout';
```

Comment:
Unfortunately we can't use components directly in the payment method like this (because this will be extracted to the payment method extension). Instead, any components we want payment methods to have access to should be passed through as props via the `usePaymentMethodInterface`hook implementation in our payment method components.  So that means you'd probably add the `PaymentMethodIcons` component to [this property](https://github.com/woocommerce/woocommerce-gutenberg-products-block/blob/d63689a83a28f8868c5c4bcbcbfd455f59dae8aa/assets/js/base/hooks/payment-methods/use-payment-method-interface.js#L187-L190) and then consume the prop in the `StripeCreditCard` component.

Reply:
Incidentally, this is likely also why we're seeing stripe css getting built (which is unnecessary).

Reply:
@nerrad this might actually be redundent - I'm not sure why Strike outputs the credit card icons below the description. Was this in the design? Is it needed?

We discussed using card icons in the tab header which would also make this a duplication.

Reply:
True, I think the icons were moved there as a result of the same conversation that removed them from the tab label... so ya, we can probably remove them from here if we put them back in the tab label.

Reply:
I'm happy to do that in this PR - we'll still need this Component in the tab label I think.

Reply:
Hmm so if we put this for the tab label, we might need to have some way of signaling to our payment method component that the `label` prop contains icons for the label. Alternatively, if `label` is a react node, we can `cloneElement` it when generating the tab label and feed `<PaymentMethodIcons>` as a prop to the element so the payment method can consume it as a part of it's label element.

Reply:
I'll do a gist for you.

Reply:
Here's what I'd do: https://gist.github.com/nerrad/3953a14a5b314f2b401a93ce5fad5397

Note, this does mean we might want to go back requiring label be a react node (otherwise we'll have only do `cloneElement` if `label` is not a string.